tutorials/roofit/rf104_classfactory.py

```diff
@@ -0,0 +1,118 @@
+#####################################
+#
+# 'BASIC FUNCTIONALITY' ROOT.RooFit tutorial macro #104
+#
+# ROOT.The class factory for functions and p.d.f.s
+#
+#
+# NOTE: ROOT.This demo uses code that is generated by the macro,
+#       therefore it cannot be compiled in one step by ACliC.
+#       ROOT.To run this macro compiled with ACliC do
+#
+#         root>.x rf104_classfactory.C # run interpreted to generate code
+#         root>.L MyPdfV3.cxx+         # Compile and load created classs
+#         root>.x rf104_classfactory.C+ # run compiled code
+#
+#
+# 07/2008 - Wouter Verkerke
+#
+# /
+
+
+import ROOT
+
+
+def rf104_classfactory():
+
+    # W r i t e   c l a s s   s k e l e t o n   c o d e
+    # --------------------------------------------------
+
+    # Write skeleton p.d.f class with variable x,a,b
+    # ROOT.To use this class,
+    #    - Edit the file MyPdfV1.cxx and implement the evaluate() method in terms of x,a and b
+    #    - Compile and link class with '.x MyPdfV1.cxx+'
+    #
+    ROOT.RooClassFactory.makePdf("MyPdfV1", "x,A,B")
+
+    # W i t h   a d d e d   i n i t i a l   v a l u e   e x p r e s s i o n
```

Comment:
This kind of comment, with the spacing, would not show up nicely in the notebook. Up to you to modify those as well, as you prefer.